docs/mining/proof-of-work/pool-mining/gominer/linux.md

```diff
@@ -0,0 +1,99 @@
+# **<i class="fa fa-linux"></i> *gominer* Linux Pool Mining User Guide**
+
+This guide is intended for basic initial setup of gominer on a mining pool of choice using a configuration file, more advanced options are available.
+
+Last updated on April 28th, 2017.
+
+---
+
+### **<i class="fa fa-download"></i> Download and Extract the *gominer* Binary**
+
+
+>Download .tar.gz File
+
+Visit [https://github.com/decred/decred-binaries/releases](https://github.com/decred/decred-binaries/releases) to download the latest gominer Linux binaries. Only 64 bit versions of gominer are available. The current latest releases are:
+
+- `gominer-linux-amd64-cuda-v1.0.0.tar.gz`,
+- `gominer-linux-amd64-opencl-v1.0.0.tar.gz`,
+- `gominer-linux-amd64-opencladl-v1.0.0.tar.gz`.
+
+For NVIDIA graphics cards, download the `*-cuda-*.tar.gz` file. <br />
+For AMD graphics cards from the Radeon and FirePro lines, download the `*-opencladl-*.tar.gz` file. <br />
+For other graphics cards, download `*-opencl-*.tar.gz` file.
+
+For you command line lovers, here are the direct links for v1.0.0:
+
+```no-highlight
+$ wget https://github.com/decred/decred-binaries/releases/download/v1.0.0_gominer/gominer-linux-amd64-cuda-v1.0.0.tar.gz
+$ wget https://github.com/decred/decred-binaries/releases/download/v1.0.0_gominer/gominer-linux-amd64-opencl-v1.0.0.tar.gz
+$ wget https://github.com/decred/decred-binaries/releases/download/v1.0.0_gominer/gominer-linux-amd64-opencladl-v1.0.0.tar.gz
+```
+
+>Extract the .tar.gz File
+
+Extract or copy all of the files to a new directory. For this tutorial we will use `~/decred/` as our working directory. From the command line:
+
+```no-highlight
+$ tar -xf gominer-linux-amd64-cuda-v1.0.0.tar.gz -C ~/decred/
+```
+
+Substitute the filename for the one you downloaded and the directory for where you want it installed.  When you unpack the tar, it will create a directory named `gominer-linux-amd64-opencladl-v1.0.0`. The contents of this directory should be `blake256.cl`, `blake256-old.cl`, `gominer`, `LICENSE`, `README.md`, and `sample-gominer.conf`.
+
+For the CUDA version, the CUDA runtime library is now included, so you no longer need to download the 1GB SDK from Nvidia.  The files in this version will also include `libcudart.so.8.0` and `rungominer.sh`. The script is used to start gominer with the included runtime library for your convenience.  You must have drivers installed for your graphics card that support CUDA 8, however.  Older drivers will cause gominer to crash on launch.
```

Comment:
Just say the CUDA runtime is included and ditch what is after the comma (since that will be weird a few version in to this).